coldfront/plugins/freeipa/README.md

```diff
@@ -62,30 +62,21 @@ $ sss_cache -E
 
```

Comment:
Under Design section:  FreeIPA unix groups can be set on a per allocation basis using using a allocation attribute named "freeipa_group" - remove double 'using' and change to 'using an allocation'

```suggestion
add 'active' to allocation info as these tasks only happen to active allocations.  Might want to also explain users are removed from freeipa group(s) when allocation expires.
```